### PR TITLE
Default to empty dict if no components in status.yml.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: app/worker.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def handler(event, context):
         except IOError:
             current_status = dict()
 
-    current_components = current_status['components']
+    current_components = current_status.get('components', {})
```
Comment (johngian) [inline app/worker.py:45]: Does `yaml.load()` always return a `dict` based on your input? A `yaml` structure might also return a list and this line is going to fail.

**Comment by johngian** [comment]
r+wc

File: app/worker.py (hunk 1 above)
Comment (glogiotatidis) [reply app/worker.py:45]: Yes b/c of the structure of status.yml that this code generates.

**Comment by glogiotatidis** [comment]
Thanks!

**Pull request merged**